packages/atom/src/atom.ts

```diff
@@ -1,14 +1,18 @@
 import { EventEmitter } from "events";
 import { Operation } from "effection";
 import { on } from '@effection/events';
+import { Subscribable, SymbolSubscribable } from '@effection/subscription';
 import { Slice } from "./slice";
 
-export class Atom<S> {
+export class Atom<S> implements Subscribable<S,void> {
   initial: S;
   state: S;
 
   subscriptions = new EventEmitter();
 
+  states = Subscribable.from(on(this.subscriptions, 'state'))
+    .map(([state]) => state as S)
+
```

Comment:
This is like the perfect use case for channels, see https://github.com/thefrontside/effection/pull/144.